TestSamples/.swiftlint.yml

```diff
@@ -0,0 +1,4 @@
+parent_config: ../.swiftlint.yml
+
+disabled_rules:
+  - force_unwrapping
```

Comment:
`l`: I think we currently don't force unwrap in the TestSamples. We could remove this and even this file. If we once need to force unwrap, we can always add this.
```suggestion
```

Reply:
We force unwrap here https://github.com/getsentry/sentry-cocoa/blob/83b66da4b983e58a75879d4342279ad02be91028/TestSamples/SwiftUITestSample/SwiftUITestSample/SwiftUITestSampleApp.swift#L50

Reply:
You can also disable the swift_lint only for specific section of the code with 
```
// swiftlint:disable force_unwrapping
// Explain why you need to force unwrap.

// swiftlint:enable force_unwrapping
```

Reply:
Thank you, yeah, that makes more sense in this case.